packages/skin/src/routes/_index/component/card/+page.marko

```diff
@@ -0,0 +1,566 @@
+import imgCards from "/src/routes/static/img/cards-card.jpeg";
+
+<div id="card">
+    <section-header metadata=metadata/>
+
+    <p>
+        Card is a navigation component that can be used to display content in a structured and visually appealing way. It can contain various elements such as images, text, buttons, and more. Cards are commonly used in dashboards, product listings, and other areas where content needs to be presented in a compact format and allow the user to navigate to related content, whether on-screen or a different URL.
+    </p>
+
+    <h2 id="card-vertical">Vertical Card (default)</h2>
+
+    <p>
+        The default card is a vertical card with a container background, a header, content, and footer.
+    </p>
+
+    <p>
+        The initial card variant is an internally-actionable card - the card itself is not clickable, but it contains an action button inside that can be clicked to perform an action.
+    </p>
+
+    <p>
+        Unless the user is taken to another page entirely, when clicking on the action button for optimal accessibility, the button should get an <highlight>aria-pressed="true"</highlight> attribute added either real-time or after postback of the same interface. This provides clear feedback to the user that the action has been triggered, especially for screen readers and assistive technologies.
+    </p>
+
+    <p>
+        This example includes the most essential aspects of a card - header with a hero image, body with content, and a footer with an action button. The vertical card is by default a square (1:1 aspect ratio) card, but you can also change the aspect ratio by adding a modifier class such as <highlight>.card--aspect-5-4</highlight> or <highlight>.card--aspect-16-9</highlight> to the card container. This will trigger the card to have a 5:4 and 16:9 aspect ratios respectively.
+    </p>
+
+<demo>
+    <div style="display: flex; gap: 24px; flex-wrap: wrap;">
+        <span class="card">
+            <div class="card__media">
+                <img src=imgCards alt="Card Hero Image" class="card__hero-image">
+            </div>
+            <div class="card__main">
+                <div class="card__description">
+                    Wayne Gretzky's rookie card is so valuable that collectors keep it in climate-controlled vaults guarded by trained penguins.
+                </div>
+                <div class="card__footer">
+                    <button class="btn btn--primary btn--fluid">See Details</button>
+                </div>
+            </div>
+        </span>
+
+        <span class="card card--aspect-5-4">
```

Comment:
I think i'd prefer to separate out the aspect ratio modifiers into their own examples. I prefer having one example per code block if we can.